src/Weasel.Postgresql.Tests/Tables/reading_existing_table_from_database.cs

```diff
@@ -75,6 +75,7 @@ public async Task read_multi_pk_fields()
         table.AddColumn<string>("first_name");
         table.AddColumn<string>("last_name");
 
+        await DropSchemaObjectInDatabase(table);
```

Comment:
**QUESTION:** Why do we need to drop schema objects if we already reset schema a few lines above?

Reply:
It solved some flaky tests for me. Now I see that the tests work without this additional drops. Let me remove it and run tests here afterwards, maybe it's not really necessary.

Reply:
They are removed, please run tests here.